plugins/powerd/fu-plugin-powerd.c

```diff
@@ -109,7 +113,9 @@ fu_plugin_update_prepare (FuPlugin *plugin,
 			     "unless forced ");
 		return FALSE;
 	}
-	if (current_level < fu_context_get_battery_threshold (ctx)) {
+	if (fu_context_get_battery_threshold (ctx) != FU_BATTERY_VALUE_INVALID &&
```

Comment:
This is not formatted with clang-format... Run `./contrib/setup.py` as explained in CONTRIBUTING.md and amend this commit